extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/session/TransactionScopedSession.java

```diff
@@ -52,10 +52,8 @@
 import org.hibernate.query.criteria.JpaCriteriaInsertSelect;
 import org.hibernate.stat.SessionStatistics;
 
-import io.quarkus.arc.Arc;
-import io.quarkus.hibernate.orm.runtime.RequestScopedSessionHolder;
-import io.quarkus.runtime.BlockingOperationControl;
-import io.quarkus.runtime.BlockingOperationNotAllowedException;
+import java.util.List;
+import java.util.Map;
 
 public class TransactionScopedSession implements Session {
```

Comment:
Sorry, I forgot about that, but noticed just now: we'd need the same thing in `io.quarkus.hibernate.orm.runtime.session.TransactionScopedStatelessSession`.